src/main/java/ru/yandex/practicum/filmorate/storage/dao/film/DirectorDao.java

```diff
@@ -0,0 +1,27 @@
+package ru.yandex.practicum.filmorate.storage.dao.film;
+
+import ru.yandex.practicum.filmorate.model.Director;
+import ru.yandex.practicum.filmorate.model.Film;
+
+import java.util.List;
+
+public interface DirectorDao {
+
+    Director createDirector(Director director);
+
+    Director updateDirector(Director director);
+
+    List<Director> findAllDirectors();
```

Comment:
Некритично, но для единого стиля именования методов проекта рекомендую использовать **get** вместо **find**.
```suggestion
    List<Director> getAllDirectors();
```